tests/drop_test.py

```diff
@@ -0,0 +1,133 @@
+import random
+from typing import NoReturn
+
+from lynx.common.actions.action import Action
+from lynx.common.actions.drop import Drop
+from lynx.common.object import Object
+from lynx.common.player import Player
+from lynx.common.scene import Scene
+from lynx.common.vector import Vector
+from lynx.common.actions.update_resources import UpdateResources
+
+class TestDropSerialization:
+    expected_serialization_drop = '{"type": "Drop", "attributes": {"object_id": 1, "target_position": {"x": 1, "y": 0}}}'
+
+    def test_success_serialization(self) -> NoReturn:
+        serialized_drop = Drop(object_id=1, target_position=Vector(1, 0)).serialize()
+
+        assert self.expected_serialization_drop == serialized_drop
+
+    def test_success_deserialization(self):
+        expected_drop = Drop(object_id=1, target_position=Vector(1, 0))
+        dummy_drop = Drop.deserialize(self.expected_serialization_drop)
+
+        assert dummy_drop == expected_drop
+
+class TestDropApply:
+    def test_success_apply(self):
```

Comment:
Please explain test scenarios in names more throughly e.g.
`test_drop_single_resource_in_overworld_creates_single_object`